### DISCUSSION TITLE
Add initializetestapp() like firebase-rules-testing

### DISCUSSION THREAD

**Discussion by warrendodsworth** [angular/angularfire#2844] [Ideas]
Have a feature request?
========================
How will it make current
workarounds straightforward? What potential bugs and edge cases does it help to avoid?

Add initializetestapp() to AngularFireModule
To enable auth mocking, like they do with the firebase-rules-testing initialization. 

Currently need to disable rules to test angular services with angular fire.

[a comment by an automated account is not shown]

**Comment by jamesdaniels** [comment]
Moving to discussions.

Our current recommendation is to use the emulators entirely in AngularFire, as swapping out AngularFireModule for initializeTestApp is not trivial (and likely not possible with the current AF API).

Including auth https://firebase.google.com/docs/emulator-suite/connect_auth
